docs/how-to-guides/scaling-feast.md

```diff
@@ -0,0 +1,27 @@
+# Scaling Feast
```

Comment:
if you want this to appear in the how-to guides section, you'll need to add it to SUMMARY.md

same for structuring-repos.md

Reply:
Righto